prowler/compliance/aws/aws_account_security_onboarding_aws.json

```diff
@@ -0,0 +1,445 @@
+      },
+      {
+        "Id": "Logging",
+        "Name": "Logging",
+        "Description": "Logging in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "DNS-Resolver-Queries-SIEM",
+            "Name": "Send DNS resolvers queries to SIEM"
+          },
+          {
+            "Section": "VPC-Flow-Logs",
+            "Name": "Send VPC flow logs (only denys) to S3 bucket"
+          },
+          {
+            "Section": "S3-Logs-Critical-Buckets",
+            "Name": "Send S3 access logs for critical buckets to separate S3 bucket"
+          },
+          {
+            "Section": "ALB-CFD-To-SIEM",
+            "Name": "Establish ready-to-be-enabled pipelines to deliver ALB and CFD to SIEM to toggle in case of emergency and investigations"
+          }
+        ],
+        "Checks": [
+            "cloudtrail_logs_s3_bucket_access_logging_enabled",
+            "cloudtrail_s3_dataevents_write_enabled",
+            "cloudtrail_cloudwatch_logging_enabled"
+        ]
+      },
+      {
+        "Id": "Alerting",
+        "Name": "Alerting",
+        "Description": "Enable alerts in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "Guardutty",
+            "Name": "GuardDuty"
+          },
+          {
+            "Section": "Cloudtrail",
+            "Name": "Cloudtrail"
+          },
+          {
+            "Section": "Cost-Anomaly",
+            "Name": "Cost-Anomaly"
+          },
+          {
+            "Section": "SecurityHub",
+            "Name": "SecurityHub"
+          },
+          {
+            "Section": "WAF",
+            "Name": "WAF"
+          },
+          {
+            "Section": "Shield",
+            "Name": "Shield"
+          },
+          {
+            "Section": "R53-DNS-Resolver",
+            "Name": "R53-DNS-Resolver"
+          },
+          {
+            "Section": "Vulnerability-Scanning",
+            "Name": "Vulnerability-Scanning"
+          },
+          {
+            "Section": "Prod",
+            "Name": "Prod"
+          },
+          {
+            "Section": "QA",
+            "Name": "QA"
+          }
+        ],
+        "Checks": [
+          "guardduty_is_enabled",
+          "guardduty_no_high_severity_findings",
+          "route53_domains_privacy_protection_enabled",
+          "securityhub_enabled",
+          "elbv2_waf_acl_attached",
+          "apigateway_restapi_waf_acl_attached",
+          "shield_advanced_protection_in_associated_elastic_ips",
+          "shield_advanced_protection_in_classic_load_balancers",
+          "shield_advanced_protection_in_cloudfront_distributions",
+          "shield_advanced_protection_in_global_accelerators",
+          "shield_advanced_protection_in_internet_facing_load_balancers",
+          "shield_advanced_protection_in_route53_hosted_zones",
+          "ecr_repositories_scan_images_on_push_enabled",
+          "ecr_repositories_scan_vulnerabilities_in_latest_image"
```

Comment:
These checks do not apply here, the alerting checks are in the CloudWatch service.

Reply:
Done!